packages/jamtools/features/modules/dashboards/keytar_and_foot_dashboard/keytar_and_foot_dashboard.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import {ModuleAPI} from '~/core/engine/module_api';
+
+import {MultiOctaveSupervisor} from './multi_octave_supervisor';
+
+const KeytarAndFootDashboard = async (moduleAPI: ModuleAPI, dashboardName: string) => {
+    const multiOctaveSupervisor = new MultiOctaveSupervisor(moduleAPI, dashboardName + '|multi-octave');
+    await multiOctaveSupervisor.initialize();
+
+    moduleAPI.registerRoute(dashboardName, {}, () => (
```

Comment:
The string passed into this is automatically namespaced to the current module. I had the caller of `KeytarAndFootDashboard` pass this in just so the control of all route names for the dashboard module can be centralized atm. Once we make dashboards a first-class object in the system, then the name will be encapsulated in the given dashboard's code itself.